installers/kubectl/client-setup.sh

```diff
@@ -12,41 +12,49 @@
 #  limitations under the License.
 
 # This script should be run after the operator has been deployed
-export PGO_OPERATOR_NAMESPACE=${PGO_OPERATOR_NAMESPACE:-pgo}
+export PGO_OPERATOR_NAMESPACE="${PGO_OPERATOR_NAMESPACE:-pgo}"
+PGO_USER_ADMIN="${PGO_USER_ADMIN:-pgouser-admin}"
 
 # Check that the pgouser-admin secret exists
-if [ -z "$(kubectl get secret -n $PGO_OPERATOR_NAMESPACE pgouser-admin)" ]
+if [ -z "$(kubectl get secret -n ${PGO_OPERATOR_NAMESPACE} ${PGO_USER_ADMIN})" ]
 then
-    echo "pgouser-admin Secret not found in namespace: $PGO_OPERATOR_NAMESPACE"
+    echo "${PGO_USER_ADMIN} Secret not found in namespace: ${PGO_OPERATOR_NAMESPACE}"
     echo "Please ensure that the PostgreSQL Operator has been installed."
     echo "Exiting..."
     exit 1
 fi
 
 # Check that the pgo.tls secret exists
-if [ -z "$(kubectl get secret -n $PGO_OPERATOR_NAMESPACE pgo.tls)" ]
+if [ -z "$(kubectl get secret -n ${PGO_OPERATOR_NAMESPACE} pgo.tls)" ]
```

Comment:
If we are adding a variable for the user secret why not add it for the pgo.tls secret as well?

Reply:
`pgo.tls` is unchanging, whereas there may be multiple users accessing an Operator :wink:

Reply:
gotcha